NanoAnalysis/python/nanoZZ4lAnalysis.py

```diff
@@ -309,9 +315,10 @@
                           'keep HTXS_Higgs*',
                           'keep HTXS_njets30',
                           'keep Pileup*',
+                          'keep GenJet_*',
+                          'keep LHEMela*', 
                           'keep GenJet*',
```

Comment:
Since you added 'keep GenJet_*' above, can you remove this one (that will keep also GenJetAK8, which I think is not needed)